index.html

```diff
@@ -68,6 +64,216 @@ <h2 class="connect">LET’S CONNECT</h2>
+                <li class="work__item">Back End Dev</li>
+                <li class="work__item">2015</li>
+              </ul>
+            </div>
+            <p>
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
+        <article class="work_card">
+          <header class="work__header">
+            <img
+              src="assets/snapshoot3.png"
+              alt="Snapshoot Portfolio 3"
+              class="work__image"
+            />
+          </header>
+          <div class="portfolio__content">
+            <h3 class="portfolio__title color-dark">Tonic</h3>
+            <div class="portfolio__profile">
+              <h4 class="portfolio__author color-info">Canopy</h4>
+              <ul class="portfolio__list">
+                <li class="work__item">Back End Dev</li>
+                <li class="work__item">2015</li>
+              </ul>
+            </div>
+            <p class="portfolio__description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
+        <article class="work_card">
+          <header class="work__header">
+            <img
+              src="assets/snapshoot4.png"
+              alt="Snapshoot Portfolio 4"
+              class="work__image"
+            />
+          </header>
+          <div class="portfolio__content">
+            <h3 class="portfolio__title color-dark">Multi-Post Stories</h3>
+            <div class="portfolio__profile">
+              <h4 class="portfolio__author color-info">Canopy</h4>
+              <ul class="portfolio__list">
+                <li class="work__item">Back End Dev</li>
+                <li class="work__item">2015</li>
+              </ul>
+            </div>
+            <p class="portfolio__description">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="portfolio__tags">
+              <li class="tag">HTML</li>
+              <li class="tag">CSS</li>
+              <li class="tag">JavaScript</li>
+            </ul>
+            <div class="button_primary">
+              <a>See project</a>
+            </div>
+          </div>
+        </article>
```

Comment:
- [ ] Nice work Implementing the portfolio cards and layout, upon careful examination I notice your design is not correspond with the Figma design, some of the key things you need to implement
 - [ ] you content (margin)
 - [ ] padding 

- [ ] Your Design
![image](https://github.com/montsecruz9/personal-portfolio/assets/68823331/81b3496b-d071-4c89-ac6b-3f88c6e0b97a)

- [ ] Figma Design
![image](https://github.com/montsecruz9/personal-portfolio/assets/68823331/7ba21eed-5dbc-489e-a923-8882e6d300cb)